kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/kms/CachingKms.java

```diff
@@ -114,4 +115,20 @@ public CompletionStage<K> resolveAlias(@NonNull String alias) {
         });
         return resolved;
     }
+
+    @VisibleForTesting
+    void decryptDekCacheCleanUp() {
+        decryptDekCache.synchronous().cleanUp();
+    }
+
+    @VisibleForTesting
+    void resolveAliasCacheCleanUp() {
+        resolveAliasCache.synchronous().cleanUp();
+    }
+
+    @VisibleForTesting
+    void notFoundAliasCacheCleanUp() {
+        notFoundAliasCache.cleanUp();
+    }
```

Comment:
Do we gain anything from having three cleanup methods? Or would a single `cleanUp` method suffice?

Reply:
Yes... I was torn.  How `CachingKms` achieves its caching is a implementation concern, so exposing the three caches could be seen as breaking the abstraction.   OTOH, the test it really only concerned with a single aspect.

I went with second option as it seemed like least intrusive.    SInce you've had the same thought, I'm inclined to a singe cleanup method.

Reply:
Its primarily why I was asking about admin interfaces. I can see three methods making more sense for that and thus being a genuine part of an interface.